ipaclient/install/client.py

```diff
@@ -2165,7 +2166,18 @@ def install_check(options):
             "authentication resources",
             rval=CLIENT_INSTALL_ERROR)
 
-    # when installing with '--no-sssd' option, check whether nss-ldap is
+    # When installing without the "--no-sudo" option, check whether sudo is
+    # available.
+    if options.conf_sudo:
+        try:
+            subprocess.Popen(['sudo'])
```

Comment:
can you please run at least 'sudo -V' here? This would not start sudo in authentication mode but would at least allow to verify the tool exists.

Reply:
Thanks for the hint. Fixed.